src/main/java/net/glowstone/block/blocktype/BlockType.java

```diff
@@ -403,7 +403,33 @@ public void onEntityStep(GlowBlock block, LivingEntity entity) {
         // do nothing
     }
 
-    public void requestPulse(GlowBlockState state) {
-        // do nothing
+    /**
+     * Request pulsing to the block
+     *
+     * @param block the block to pulse
+     */
+    public void requestPulse(GlowBlock block) {
+        if (getPulseTickSpeed() != null) {
+            block.getWorld().requestPulse(block, getPulseTickSpeed(), isPulseOnce());
```

Comment:
Hmm maybe, but I don't know if that's necessary. The pulse map would have to access the ItemTable a lot more than it currently does.